masschallenge/django-accelerator
# [AC-9634]Show Enterprises on /startups and correct field label on partner forms

## [AC-9634](https://masschallenge.atlassian.net/browse/AC-9634)

**Changes Introduced**
- Show Enterprises on `/mystartups`
- Correct field labels on EditCompanyProfileForm to match mockup

**Testing**
- Check out `AC-9634`, Run `make migrate`
- As a logged-in user, visit http://localhost:8181/mycompanies/ or https://test-cam.masschallenge.org/mycompanies/
- As a Staff user, visit http://staff.localhost:8181/partner/add/ or https://test-cam.masschallenge.org/partner/add/
- Confirm that the definition of done according to [AC-9634](https://masschallenge.atlassian.net/browse/AC-9634) is satisfied

[Sibling PR](https://github.com/masschallenge/accelerate/pull/3336)
